### PR TITLE
Added Testing CI

### PR DESCRIPTION
**Pull request by KiOui**
Added CI for running tests and automatically building a docker image

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            DOCKER_TAG_PRODUCTION: docker.pkg.github.com/kioui/tosti/tosti:${{ github.sha }}
+        steps:
+            - name: "Checkout repository"
+              uses: "actions/checkout@v2"
+
+            - name: "Build Docker image"
+              run: docker build --tag "${DOCKER_LATEST}" .
+
+            - name: "Login to GitHub"
+              run: echo "${{ secrets.GH_TOKEN }}" | docker login docker.pkg.github.com -u KiOui --password-stdin
```
Comment (JobDoesburg) [inline .github/workflows/docker.yaml:23]: I would say use a secret for the username as well (idem for the urls above)

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            DOCKER_TAG_PRODUCTION: ${{ secrets.DOCKER_URL }}:${{ github.sha }}
+        steps:
+            - name: "Checkout repository"
+              uses: "actions/checkout@v2"
+
+            - name: "Build Docker image"
+              run: docker build --tag "${DOCKER_LATEST}" .
+
+            - name: "Login to GitHub"
+              run: echo "${{ secrets.GH_TOKEN }}" | docker login docker.pkg.github.com -u "${{ secrets.GH_USER }}" --password-stdin
```
Comment (JobDoesburg) [inline .github/workflows/docker.yaml:23]: [https://github.community/t/how-do-i-get-gh-username-based-on-actions-events/17882/3](https://github.community/t/how-do-i-get-gh-username-based-on-actions-events/17882/3) seems like you can use `github.actor` which is way better

File: docker-compose.yml.example [hunk 3]
```diff
@@ -28,7 +28,7 @@ services:
           - backend-django-tosti
 
     tosti-django:
-        image: "larsvanrhijn/tosti:latest"
+        image: "docker.pkg.github.com/kioui/tosti/tosti:latest"
```
Comment (JobDoesburg) [inline docker-compose.yml.example:31]: In the docker.yaml file, you reference it as `docker.pkg.github.com/${GITHUB_REPOSITORY}/${GITHUB_REPOSITORY##*/}:latest`, but shouldn't it be the repository owner for the first one

Comment (KiOui) [reply docker-compose.yml.example:31]: Nope, check the latest docker image in the repo and the latest succeeded docker actions build. GITHUB_REPOSITORY outputs KiOui/TOSTI

Comment (KiOui) [reply docker-compose.yml.example:31]: And ${GITHUB_REPOSITORY##*/} outputs TOSTI

**Pull request merged**